src/mds/Server.cc

```diff
@@ -462,9 +464,12 @@ void Server::reclaim_session(Session *session, const cref_t<MClientReclaim> &m)
   if (flags & CEPH_RECLAIM_RESET) {
```

Comment:
@gregsfortytwo -- @dparmar18 mentioned discussing with you to remove this check. What's the reason for that?

Reply:
@vshankar I remember discussing where he didn't tell me to remove it  but he was okay with it If I removed it

Reply:
I think it's better to keep this check and NOT do `ceph_abort()`, instead send message to client that this is not allowed. What do you think? I've pushed this change if you want to have a look